using_openshift/authentication.adoc

```diff
@@ -0,0 +1,28 @@
+= Authentication
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Web Console authentication
+
+The web console automatically redirects to `/oauth/authorize` to obtain a token to make API calls.
```

Comment:
This being out of context, might want to make clear that this URI is against the master (well, assuming OAuth is at the master).